src/Contract/Operation/Applyable.php

```diff
@@ -20,9 +21,9 @@ interface Applyable
     /**
      * Execute a callback for each element of the collection.
      *
-     * @param callable(TKey, T):bool ...$callables
+     * @param callable(T, TKey, Iterator<TKey, T>): bool ...$callbacks
```

Comment:
the interface was out of sync with the [operation](https://github.com/loophp/collection/pull/104/files#diff-79d43bca592b76fa0fb2583de4ea453a994e67ea88ab8d27e403579525c8c834R31)

Reply:
Maybe we should fix that in another PR, wdyt?

Reply:
well it's causing an issue with PHPStan in this PR so that's why I updated it here 😄 

It's related to [this error which was ignored](https://github.com/loophp/collection/pull/104/files#diff-22fe68f6ec4412b512a01a6526b11f8ad2f9e9c07ebcd32d018a131037059de7L14-L16). In PHP 8 the parameter name for `mb_strtolower` was changed to from `$str` to `$string`, so there would be x2 errors now depending on which PHP version PHPStan is run. I thought I'd just go ahead and fix it :)

Reply:
Ok.

Reply:
I made a commit on `master` to fix it, you can rebase your branch if you want.